src/common/store/pages/explore/exploreCollections/selectors.ts

```diff
@@ -6,8 +6,7 @@ import { AppState } from 'store/types'
 
 import { ExploreCollectionsVariant } from '../types'
 
-const getBaseState = (state: AppState) =>
-  state.application.pages.exploreCollections
+const getBaseState = (state: AppState) => state.pages.exploreCollections
```

Comment:
This should be updated from `AppState` to `CommonState`, otherwise the dependency on `AppState` creates a dependency on pretty much everything in the client store